src/frontend/src/lib/styles/global/gix.scss

```diff
@@ -171,9 +175,22 @@ div.modal {
 		flex-direction: column;
 		min-height: 100%;
 
+		&.min-h-auto {
+			min-height: auto;
+		}
+
 		> div.stretch {
 			flex: 1;
-			margin: 0 0 var(--padding-3x);
+
+			margin: 0 0 var(--padding-0_5x);
+
+			&:has(+ *) {
```

Comment:
I am not sure, but doesn't the new version of `SASS` says that this must be after the other "direct" declarations? Or it is not flagged for `svelte` files?

Reply:
No warnings 🤷‍♂️. I moved it anyway, you never know.